global.json

```diff
@@ -1,8 +1,8 @@
 ﻿{
     "sources": [ "src", "test" ],
     "sdk": {
-            "architecture": "x86",
+      "architecture": "x86",
       "runtime": "clr",
-      "version": "1.0.0-rc1-update1"
+      "version": "1.0.0-rc1-update2"
```

Comment:
This version corresponds to the dnx runtime. They have updated the run time version, and I referred to the latest available one.